pandas/io/parsers/readers.py

```diff
@@ -1710,6 +1710,10 @@ def _make_engine(
             assert self.handles is not None
             f = self.handles.handle
 
+        elif not engine == "python":
```

Comment:
nitpick "!="?

Reply:
Thx, no idea why I was using the not syntax